hawkbit-dmf/hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AbstractAmqpErrorHandler.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2021 Bosch.IO GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.amqp;
+
+import org.springframework.amqp.AmqpRejectAndDontRequeueException;
+
+/**
+ * An abstract error handler for errors resulting from AMQP.
+ */
+public abstract class AbstractAmqpErrorHandler<T> implements AmqpErrorHandler{
+
+    @Override
+    public void doHandle(Throwable throwable, AmqpErrorHandlerChain chain) {
+        final Throwable cause = throwable.getCause();
```

Comment:
please add a comment why you check the `cause` here

Reply:
👍